example/lambda.json

```diff
@@ -11,5 +11,13 @@
     "/*.pyc"
   ],
   "timeout": 30,
-  "memory": 512
+  "memory": 512,
+  "vpc": {
```

Comment:
Yeah if you are using the one in test/configs as the test with vpc then you can probably just use this one as your non vpc test.